lms/djangoapps/course_api/blocks/forms.py

```diff
@@ -138,7 +179,6 @@ def _verify_no_user(requesting_user, cleaned_data, course_key):
             )
 
         # return None for user
-        return None
```

Comment:
Why remove the `return` but not the comment?

Reply:
I guess a return None is implicit. I don't quite remember the reason, perhaps it was due to a linting issue? 
Will add it back and see.

Reply:
@stvstnfrd I'm specifically talking about the platform's linting rules. I completely agree that an explicit None is beetter in many cases. 

I can add it back with an exception?